pkg/distributor/forwarding/forwarding.go

```diff
@@ -111,6 +113,11 @@ func NewForwarder(cfg Config, reg prometheus.Registerer, log log.Logger, limits
 			Name:      "distributor_forward_samples_total",
 			Help:      "The total number of samples the Distributor forwarded.",
 		}),
+		histogramsTotal: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Namespace: "cortex",
+			Name:      "distributor_forward_histograms_total",
```

Comment:
note to self: missed this, get rid of this metric and use "type" label on "distributor_forward_samples_total"